scripts/amd/gemm/tune_gemm.py

```diff
@@ -218,8 +238,7 @@ def try_config_{configStr}(M, N, K, am, ak, bk, bn, cm, cn):
 
 
 def generated_kernel_name(M, N, K, gpu_id):
-    path = os.path.dirname(os.path.abspath(__file__))
-    return f"{path}/generated_kernel{M}-{N}-{K}-{gpu_id}.py"
+    return f"generated_kernel{M}-{N}-{K}-{gpu_id}.py"
```

Comment:
Why do you change this part back? @yiqian1 added it so that when we call tune_gemm in another dir, the generated_kernel is still in `scripts/amd/gemm`.

Reply:
I was not aware the changes here, maybe there are something when I do rebase. I will revert the changes here.